networks/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Networks
```

Comment:
This seems to unnecessarily duplicate this: https://github.com/ava-labs/subnet-evm/blob/master/networks/README.md